gammapy/spectrum/tests/test_spectrum_fit.py

```diff
@@ -10,33 +10,63 @@
 from ...spectrum.spectrum_fit import SpectrumFit
 from ...utils.testing import requires_dependency, requires_data, SHERPA_LT_4_8
 from astropy.utils.compat import NUMPY_LT_1_9
-
+import logging
 
 @pytest.mark.skipif('NUMPY_LT_1_9')
 @pytest.mark.skipif('SHERPA_LT_4_8')
 @requires_dependency('sherpa')
 @requires_data('gammapy-extra')
 def test_spectral_fit():
+
+    logging.basicConfig(level=logging.INFO)
```

Comment:
I'm not sure this is a good idea.
There's probably better ways to configure the logger when running tests.

Please let's not start using loggers this way for tests in this PR, i.e. remove this from this PR.
Loggers are global and complicated and who knows how this interacts with pytest.

It's of course OK to use this locally while debugging the tests.

If you really think it's useful to have this in `master` and for all users and devs, please open a separate issue explaining why and we'll investigate / discuss if that is a good practice.